src/index.ts

```diff
@@ -27,16 +27,21 @@ class GooglePhotosExif extends Command {
       description: 'Directory into which the processed output will be written',
       required: true,
     }),
+    errorDir: flags.string({
+      char: 'd',
```

Comment:
```suggestion
      char: 'e',
```

The `d` char is already used by the outputDir flag, so this would clash - I suggest we make this `e` and also change the `char` property for `inputDir` to `i` and `outputDir` to `o` since I suspect those chars were an oversight that I copied and pasted from somewhere